google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPoller.java

```diff
@@ -213,6 +212,11 @@ void pollForChanges(@NotNull final CloudDebugProcessState state) {
     }
     catch (IOException ex) {
       LOG.warn("exception listing breakpoints", ex);
+      handleBreakpointQueryError(state, ex);
+      return;
+    } catch (Exception e) {
```

Comment:
Right now the IllegalStateException from queryServerForBreakpoints(...), but I assumed if something else happens we'd like to know about it instead of silently ignoring it and not giving the user any feedback.